pypif/obj/common/property.py

```diff
@@ -45,8 +46,10 @@ def __init__(self, name=None, scalars=None, vectors=None, matrices=None, files=N
         self.licenses = licenses
         self._conditions = None
         self.conditions = conditions
-        self._method = None
-        self.method = method
+        self._methods = None
+        self.methods = methods
+        if 'method' in kwargs:
+            self.methods = method
```

Comment:
since "methods" is the preferred name, it should come after method. That way it is always used if present.